content/blog/2019/custom-ingress-gateway/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Deploy a custom ingress gateway using cert-manager
+description: Describes how to deploy a custom ingress gateway using cert-manager manually.
+subtitle: Custom ingress gateway
+publishdate: 2019-01-08
+weight: 89
+keywords: [ingress,traffic-management]
+attribution: Julien Senon
+---
+
+This post provides instructions to manually create a custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/setup/).
+* Setup `cert-manager` with helm [chart](https://github.com/helm/charts/tree/master/stable/cert-manager#installing-the-chart)
+* We will use `demo.mydemo.com` for our example,
+  it must be resolved with your DNS
+
+## Configuring the custom ingress gateway
+
+1. Check if [cert-manager](https://github.com/helm/charts/tree/master/stable/cert-manager) was installed using Helm with the following command:
+
+    {{< text bash >}}
+    $ helm ls
+    {{< /text>}}
```

Comment:
I think this is causing the circleci errors. Missing space.
```
{{< /text >}}
```

Reply:
Yep, that would do it. I really should improve the regex in scripts/lint_site.sh to deal with this case correctly